src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -316,14 +316,14 @@ private ValueTask<FormulaValue> VisitBinaryOpNode(BinaryOpNode node, SymbolConte
                 case BinaryOpKind.GeqTime:
                     return OperatorGeqTime(this, context, node.IRContext, args);
                 case BinaryOpKind.DynamicGetField:
-                    return OperatorDynamicGetField(node, args);
+                    return new ValueTask<FormulaValue>(OperatorDynamicGetField(node, args));
 
                 default:
                     return new ValueTask<FormulaValue>(CommonErrors.UnreachableCodeError(node.IRContext));
             }
         }
 
-        private static async ValueTask<FormulaValue> OperatorDynamicGetField(BinaryOpNode node, FormulaValue[] args)
+        private static FormulaValue OperatorDynamicGetField(BinaryOpNode node, FormulaValue[] args)
```

Comment:
>OperatorDynamicGetField [](http://example.com/codeflow?start=36&length=23)

Not related to your changes, but all other operators are async functions (using the error wrapper) - we should make this one as well (and simplify the blank/error handling on it. Can be dealt with in a separate PR